### PR TITLE
CEDS-5800 add stubbed data to check missing status

### PR DESCRIPTION
**Pull request by glidester**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,12 @@ If LRN starts with:
 - 'D' - Stub will send an 'Accepted' and an 'Additional Documents Required' notifications
 - 'G' - Stub will send an 'Accepted' notification
 - 'Q' - Stub will send a 'Query Notification Message' notification
+- 'I' - Stub will send a 'Awaiting exit results' notification
```
Comment (udinobi) [inline README.md:23]: `Q` should be after `P`, isnt'it? :smile:

File: app/uk/gov/hmrc/customs/declarations/stub/connector/NotificationConnector.scala [hunk 2]
```diff
@@ -77,7 +77,7 @@ class NotificationConnector @Inject() (http: HttpClient, generator: Notification
       }
     }
 
-  private val validPrompts = List('B', 'C', 'D', 'G', 'Q', 'R', 'U', 'X')
+  private val validPrompts = List('B', 'C', 'D', 'G', 'Q', 'I', 'J', 'K', 'L', 'N', 'P', 'R', 'U', 'X')
```
Comment (udinobi) [inline app/uk/gov/hmrc/customs/declarations/stub/connector/NotificationConnector.scala:80]: Same here

**Review by udinobi — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (glidester) [reply README.md:22]: Inserted in wrong place, good catch :+1:

File: app/uk/gov/hmrc/customs/declarations/stub/connector/NotificationConnector.scala (hunk 2 above)
Comment (glidester) [reply app/uk/gov/hmrc/customs/declarations/stub/connector/NotificationConnector.scala]: :+1:

**Review by udinobi — APPROVED** [review]

**Pull request merged**